Word-to-Image-conversion/Fallback-fonts-for-custom-range/.NET/Fallback-fonts-for-custom-range/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.Office;
+
+namespace Fallback_fonts_for_custom_range
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Opens the file as stream.
+            using FileStream inputStream = new FileStream(@"../../../Template.docx", FileMode.Open, FileAccess.Read);
+            //Loads an existing Word document file stream.
+            using WordDocument wordDocument = new WordDocument(inputStream, Syncfusion.DocIO.FormatType.Docx);
+            //Adds fallback font for "Arabic" custom unicode range.
```

Comment:
@LokeshBaskar - Please modify the `custom` to `specific`

Reply:
Modify the custom to specific